runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslatorTest.java

```diff
@@ -229,6 +238,126 @@ private static DataflowPipelineOptions buildPipelineOptions() throws IOException
     return options;
   }
 
+  // Test that the transform names are what we expect them to be
```

Comment:
Can we make the comment more clear what is expected and why. Also please add a TODO comment with github issue link for more stable solution.

Reply:
Updated, plus added link to issue.